index.js

```diff
@@ -42,6 +46,7 @@ module.exports = {
     "max-line-length": [ 80, {
       ignore: "non-comments",
     } ],
+    "max-nesting-depth": 3,
```

Comment:
Excellent. If there is a common agreement on nesting level then I'm happy to adopt that too. We could also make some additions to the SUIT documentation once this and the `bem-linter` are updated. Thanks for your efforts here @dryoma, I'll try and get this merged later.